app/containers/Forms/FormCreateUser.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {Row, Col, Form, Button} from 'react-bootstrap';
+import {createFragmentContainer} from 'react-relay';
+import createUserMutation from '../../mutations/user/createUserMutation';
+
+export const FormCreateUser = props => {
+  const user = {
+    firstName: '',
+    lastName: '',
+    emailAddress: '',
+    occupation: '',
+    phoneNumber: ''
+  };
```

Comment:
You should be using the `useState` hook instead of having a mutable variable in the function like this.
Even better, use the `InputHook` component added in #102 . The `InputHook` component should be moved to the `components` directory, perhaps in a `Form` folder

Reply:
See the UserDetail.js file for an example of how the `InputHook` is used

Reply:
Removed this From and using FormEditUser with InputHook